acrobat/blocks/eventwrapper/eventwrapper.css

```diff
@@ -50,13 +50,27 @@ body:not([data-current-event]) [data-event-name=onload] {
   animation-name: fade-in;
 }
 
-
-@keyframes fade-in { 
-  0% { 
+@keyframes fade-in {
+  0% {
     opacity: 0;
   }
 
-  100% { 
+  100% {
     opacity: 1;
   }
 }
+
+@media (max-width: 600px) {
+  .wapper-rotate-pages.fsw > div > div {
+    height: auto;
+  }
+}
+
+.wapper-rotate-pages [data-test-id="lifecycle-organize-fragment"] {
```

Comment:
not ideal to reference test ids since they're part of our test infrastructure. Can you do something similar with a class pattern match?

Reply:
@peller Thank you for the comment, I fixed it. Also, I added some additional css to make it look good on smaller screens.